### ISSUE TITLE
[QA] Some NFT wearables are being replaced with base wearables

### ISSUE THREAD

**Issue by anicalbano** [decentraland/unity-explorer#1560]
### **Explorer Alpha build versions:**
v0.15.0 and older

### **Issue Description:**

Some NFT wearables are not working on alpha. Instead, they are being replaced by common wearables.

### **STR:** 

  1. To check this, we should own an NFT wearable that's not being propperly shown, and the only step to reproduce it is to equip it and see how it works.

### **Expected behaviour:**
All NFT wearables must be visible

#### **Evidence:**

![image](https://github.com/user-attachments/assets/37874efc-2ba5-415c-bcc1-9e097712e814)

**Comment by lorux0** [comment]
It was missing the AB. I triggered conversion. Is working now:

![Image](https://github.com/user-attachments/assets/b2f8e82a-6ddd-4f3f-8852-00525f11c7fe)

**Comment by anicalbano** [comment]
Re-checkend on my end and working as expected!
![image](https://github.com/user-attachments/assets/082662de-b88b-485b-ab59-756a221a2572)

**Comment by pravusjif** [comment]
For at least 2 of my wearables this still happens...

**Comment by lorux0** [comment]
> For at least 2 of my wearables this still happens...

Probably still the same problem. Can you provide the urns?

**Comment by pravusjif** [comment]
> Probably still the same problem. Can you provide the urns?

Yes! I see these logs after enabling the WEARABLE report category:

```
[WEARABLE]: Request for wearable with hash bafkreiffk27y6ynueuw4oiks4gxjebrnlqbp7huuft5xcutcrxrry4p5aq and urn urn:decentraland:matic:collections-v2:0x7429c88e33288599dfa5defb7eae49ae95ad44c5:0 failed, loading default wearable

[WEARABLE]: Request for wearable with hash bafkreialdm7xb6zcknj5cnqt5vehn43m4vfy775f3fukan5nlgxflhd3ga and urn urn:decentraland:matic:collections-v2:0x3a53afcd4f3a40953fa1217a56265909bb2f6309:0 failed, loading default wearable
```

Those must be these 2 wearables that are getting replaced for me (screenshot from previous explorer):
![Screenshot 2024-08-02 at 6 54 03 PM](https://github.com/user-attachments/assets/b792de50-3cee-4a03-bb1a-d04d85f19b4b)

![Screenshot 2024-08-02 at 6 54 38 PM](https://github.com/user-attachments/assets/42c1970b-8963-4e46-962e-1169141af53c)

**Comment by DafGreco** [comment]
Hey @lorux0 seems that initializing an avatar with the urn of Og doki shoes the shoes itself do not show up

**Comment by DafGreco** [comment]
Fixed in both platforms :
URN:
`urn:decentraland:matic:collections-v2:0x722dbe4677dd3069c1277946aab57b2e8553bd0f:0`

![Image](https://github.com/user-attachments/assets/0c5a86fa-4aeb-4072-b4f8-1518f71e220f)
